lib/checks/aria/aria-required-attr-evaluate.js

```diff
@@ -56,11 +55,19 @@ function ariaRequiredAttrEvaluate(node, options = {}, virtualNode) {
     }
   }
 
+  if (
```

Comment:
We should add a comment that gives a reason for this special case and point to the comment from Sarah in the original issue.